<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кривулька</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/app.ico</normaloff>res/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QStackedWidget" name="pager">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>275</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="solo">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QWidget" name="output" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="duo">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QWidget" name="output_l" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="output_r" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>250</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>250</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="f1">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Функция графика:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Preferred</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>ƒ(ⅹ)=ax²+b</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Preferred</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="draw">
               <property name="minimumSize">
                <size>
                 <width>78</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>78</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Чертить</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="3">
              <widget class="QFrame" name="f6">
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QCheckBox" name="split_option">
                  <property name="text">
                   <string>Включить разделённое отображение</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QFrame" name="f3">
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Значение B</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="input_b">
                  <property name="font">
                   <font>
                    <family>Cascadia Code</family>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QFrame" name="block_y">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Пересечение по Y</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="input_y">
                  <property name="font">
                   <font>
                    <family>Cascadia Code</family>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QFrame" name="f4">
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Цвет</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="color">
                  <item>
                   <property name="text">
                    <string>Черный</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Красный</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Синий</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Желтый</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Голубой</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Серый</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Белый</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Фиолетовый</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Другой</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="hashtag">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>#</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="hex">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="font">
                   <font>
                    <family>Cascadia Code</family>
                   </font>
                  </property>
                  <property name="inputMask">
                   <string>HHHHHH;-</string>
                  </property>
                  <property name="text">
                   <string>000000</string>
                  </property>
                  <property name="maxLength">
                   <number>6</number>
                  </property>
                  <property name="frame">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QFrame" name="f2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>230</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>1677215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Значение A</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="input_a">
                  <property name="font">
                   <font>
                    <family>Cascadia Code</family>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="4">
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="3">
              <widget class="QFrame" name="f5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>230</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>1677215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Пересечение по X</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="input_x">
                  <property name="font">
                   <font>
                    <family>Cascadia Code</family>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>576</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="recent">
     <property name="title">
      <string>Недавнее</string>
     </property>
     <addaction name="NONE"/>
    </widget>
    <addaction name="save"/>
    <addaction name="forget"/>
    <addaction name="separator"/>
    <addaction name="recent"/>
    <addaction name="clear"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="info">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="howto"/>
    <addaction name="license"/>
    <addaction name="about"/>
    <addaction name="separator"/>
    <addaction name="docs_ac"/>
    <addaction name="code_ac"/>
    <addaction name="installers"/>
    <addaction name="task_ac"/>
   </widget>
   <addaction name="file"/>
   <addaction name="info"/>
  </widget>
  <action name="save">
   <property name="text">
    <string>Сохранить график</string>
   </property>
  </action>
  <action name="forget">
   <property name="text">
    <string>Удалить график</string>
   </property>
  </action>
  <action name="clear">
   <property name="text">
    <string>Очистить историю</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="NONE">
   <property name="text">
    <string>[пусто]</string>
   </property>
  </action>
  <action name="howto">
   <property name="text">
    <string>Инструкция</string>
   </property>
  </action>
  <action name="license">
   <property name="text">
    <string>Лицензия</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="code">
   <property name="text">
    <string>Код</string>
   </property>
  </action>
  <action name="docs">
   <property name="text">
    <string>Документы</string>
   </property>
  </action>
  <action name="task">
   <property name="text">
    <string>Задание</string>
   </property>
  </action>
  <action name="installators">
   <property name="text">
    <string>Инсталляторы</string>
   </property>
  </action>
  <action name="docs_ac">
   <property name="text">
    <string>Документы</string>
   </property>
  </action>
  <action name="code_ac">
   <property name="text">
    <string>Код</string>
   </property>
  </action>
  <action name="installers">
   <property name="text">
    <string>Установщики</string>
   </property>
  </action>
  <action name="task_ac">
   <property name="text">
    <string>Задание</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
